.github/renovate.json5

```diff
@@ -19,6 +19,12 @@
 			"allowedVersions": "<= 3.1",
 			"matchManagers": [ "nuget" ],
 			"matchPackageNames": [ "Microsoft.Extensions.Configuration.Json", "Microsoft.Extensions.Logging.Configuration" ]
+		},
+		{
+			// TODO: <= 1.7.0-rc.1 for invalid response on monitoring endpoint, last failed version 1.8.0-rc.1
```

Comment:
I'd like a reference here to all the issues we're waiting for - and of course create them if upstream is not aware of them just yet.

Reply:
https://github.com/JustArchiNET/ArchiSteamFarm/blob/main/ArchiSteamFarm/Core/OS.cs#L55